mpsl/doc/cx.rst

```diff
@@ -1,72 +1,55 @@
 .. _mpsl_cx:
 
-IEEE 802.15.4 External Radio Coexistence
-########################################
+Short-Range Protocols External Radio Coexistence
+################################################
 
 .. contents::
    :local:
    :depth: 2
 
 The radio coexistence feature allows the application to interface with several types of packet traffic arbiters (PTAs).
-This allows for better radio performance for devices using multiple interfering radios simultaneously, like a combination of IEEE 802.15.4 and Wi-Fi.
-The PTA arbitrates the requested radio operations between all radios to avoid interference, increasing the performance of the radio protocols enabled simultaneously.
-The exact arbitration algorithm is dependent on the specific PTA used.
+PTAs arbitrate the requested radio operations between all radios to avoid interference, providing better radio performance to devices using multiple interfering radios simultaneously, like a combination of IEEE 802.15.4, Bluetooth LE, and Wi-Fi.
```

Comment:
```suggestion
PTAs arbitrate the requested radio operations between all radios to avoid interference, providing better radio performance to devices using multiple interfering radios simultaneously, like a combination of IEEE 802.15.4, Bluetooth® Low Energy (LE), and Wi-Fi.
```

Reply:
fixed